aybapi/impl/bot.py

```diff
@@ -0,0 +1,55 @@
+class Bot:
+    def __init__(self, client, info):
```

Comment:
why aren't you just using a dataclass for this?

You could use `dataclasses_json` on top of that and it would handle conversion from json to a python object for you.